src/api/Controllers/CodeListController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using BDMS.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Reflection;
+
+namespace BDMS.Controllers;
+
+[ApiController]
+[Route("api/v{version:apiVersion}/[controller]")]
+public class CodeListController : ControllerBase
+{
+    public BdmsContext Context { get; }
+
+    public CodeListController(BdmsContext context)
+    {
+        Context = context;
+    }
+
+    /// <summary>
+    /// Asynchronously gets the <see cref="Codelist"/>s, optionally filtered by <paramref name= "schema"/>.
+    /// </summary>
+    /// <param name = "schema">The schema of the codeLists to get.</param>
+    [HttpGet]
+    public async Task<IEnumerable<Codelist>> GetAsync(string? schema = null)
+    {
+        var codeLists = Context.Codelists.AsQueryable();
+
+        if (!string.IsNullOrEmpty(schema))
+        {
+            codeLists = codeLists.Where(c => c.Schema == schema);
+        }
+
+        return await codeLists.AsNoTracking().ToListAsync().ConfigureAwait(false);
+    }
+
+    /// <summary>
+    /// Asynchronously updates the <see cref="Codelist"/> corresponding to
+    /// the <paramref name="codelist"/> with the values to update.
+    /// </summary>
+    /// <param name ="codelist"> The <see cref="Codelist"/> to update.</param>
```

Comment:
PP: Unnötige Whitespaces in XML-Dokumentation entfernen
```suggestion
    /// <param name="codelist">The <see cref="Codelist"/> to update.</param>
```